app/messages/components/user-sider.tsx

```diff
@@ -0,0 +1,58 @@
+import AvatarList, { Avatar, AvatarType } from "app/components/avatar-list"
+
+export type UserSiderProps = React.ComponentProps<typeof AvatarList>
+
+function UserList({ users }) {
+  return (
+    <div>
+      {users?.map((avatar: AvatarType, i) => (
+        <div className="flex items-center first:pt-0 pt-8">
+          <Avatar className="" key={i} avatar={avatar} big></Avatar>
+          <span className="pl-3" style={{ color: "#999999" }}>
+            {avatar.name}
+          </span>
+        </div>
+      ))}
+    </div>
+  )
+}
+
+export default function UserSider({ list, handleOnClick, ...rest }: UserSiderProps) {
```

Comment:
I think this should read slider, maybe we can rename to `UserList` it doesn't matter how we show it

Reply:
I didn't get that, what are you suggesting @cherta?

Reply:
From what I searched out, a  `slider` component is something like what's shown below, I copied the terminology from ant.design. I don't think we should change it, because the `SiderContext` depends on that name, and I think it's more clear if we refer it to as a sider because it's easier to connect than a UserList (which could be almost anything)
![image](https://user-images.githubusercontent.com/40028772/105510033-f1746700-5cac-11eb-885d-d3472e98a3a1.png)

Reply:
Fair enough, if it's already used by ant I think we can stick to it